Makefile.js

```diff
@@ -209,6 +209,10 @@ function generateRuleIndexPage(basedir) {
     find(path.join(basedir, "/lib/rules/")).filter(fileType("js")).forEach(function(filename) {
         var rule = require(filename);
 
+        if (rule.meta.deprecated) {
```

Comment:
We could add it to the `categoriesData.deprecated.rules` array here instead of doing it manually in https://github.com/eslint/eslint/pull/6746/files#r72645142